charts/observability-pipelines-worker/templates/_pod.tpl

```diff
@@ -63,6 +63,10 @@ containers:
         value: {{ .Values.datadog.workerAPI.playground | quote }}
       - name: DD_OP_API_ADDRESS
         value: {{ .Values.datadog.workerAPI.address | quote }}
+      {{- if .Values.datadog.pciCompliant }}
```

Comment:
Can we add this value [here](https://github.com/DataDog/helm-charts/blob/main/charts/observability-pipelines-worker/values.yaml#L16) ?

Reply:
We can also use an alternative way to implement this:
```
- name: DD_OP_PCI_COMPLIANT
   value: {{ .Values.datadog.pciCompliant | quote }}
```
But as long as it works I like your variant more.

Reply:
damn, I'm in staging 🤦

Reply:
addressed in [ed0f887](https://github.com/DataDog/helm-charts/pull/1419/commits/ed0f887c84fe26666c98a34da0a2989eca7d8c37)